reports/report_details.py

```diff
@@ -551,6 +607,13 @@ def add_transnational_to_regions(regions):
         },
     },
     'ws_49': {
+        '2015': {
+            'name': '49',
+            'title': 'Internet - Main topics by region',
+            'desc': 'B.down of main topics in internet news by sex of news subjects',
+            'reports': ['global', 'region', 'country'],
```

Comment:
```suggestion
            'reports': ['global', 'region'],
```
Country should be removed